adding System.Reactive.Core to fix proxies

brettsam opened the pull request:
All tests passed previously b/c the package was referenced in test projects, but not in the actual product. Moving package ref.

brettsam committed:
first attempt to prove tests fail

brettsam commented:
/azp run

fabiocav commented on test/WebJobs.Script.Tests/WebJobs.Script.Tests.csproj:
```diff
@@ -158,7 +157,7 @@
     <EmbeddedResource Include="Resources\FileProvisioning\PowerShell\requirements_PSGalleryOnline.psd1" />
     <EmbeddedResource Include="Resources\FileProvisioning\PowerShell\profile.ps1">
-    <CopyToOutputDirectory>Never</CopyToOutputDirectory>
+      <CopyToOutputDirectory>Never</CopyToOutputDirectory>
```
nit: formatting issue here.

brettsam replied:
yeah, a previous formatting issue that is now fixed! (it's meant to be indented) :-)

brettsam committed:
adding reactive.core to script

brettsam commented:
/azp run

brettsam commented:
Recording depsjson failure here for future reference:

```
Previous file: D:\a_work\1\s\test\WebJobs.Script.Tests\bin\Debug\net6.0\Microsoft.Azure.WebJobs.Script.WebHost.deps.json
New file:      D:\a_work\1\s\src\WebJobs.Script.WebHost\bin\Debug\net6.0\Microsoft.Azure.WebJobs.Script.WebHost.deps.json

Changed:

Removed:

Added:
- System.Reactive.Core.dll: 3.0.6000.0/3.0.6000.0

Expected: True
Actual:   False
```

brettsam committed:
fixing depsjson
